src/Networking/v2/Params.php

```diff
@@ -33,4 +33,113 @@ public function adminStateUp()
         ];
     }
 
+    public function networkId()
+    {
+        return [
+            'type'        => self::STRING_TYPE,
+            'required'    => true,
+            'sentAs'      => 'network_id',
+            'description' => 'The ID of the attached network',
+        ];
+    }
+
+    public function ipVersion()
+    {
+        return [
+            'type'        => self::INT_TYPE,
+            'required'    => true,
+            'sentAs'      => 'ip_version',
+            'description' => 'The IP version, which is 4 or 6',
+        ];
+    }
+
+    public function cidr()
+    {
+        return [
+            'type'        => self::STRING_TYPE,
+            'required'    => true,
+            'sentAs'      => 'cidr',
+            'description' => 'The CIDR',
+        ];
+    }
+
+    public function tenantId()
+    {
+        return [
+            'type'        => self::STRING_TYPE,
+            'sentAs'      => 'tenant_id',
+            'description' => 'The ID of the tenant who owns the network. Only administrative users can specify a tenant ID other than their own. You cannot change this value through authorization policies',
```

Comment:
:+1: we should not be afraid of writing long descriptions, since it will always help the user when they're reading the reference docs. for anything longer than the standard page width, use [heredoc syntax](http://php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc)